Composer/packages/client/src/pages/publish/BotStatusList.tsx

```diff
@@ -244,6 +264,30 @@ export const BotStatusList: React.FC<BotStatusListProps> = ({
       },
       isPadded: true,
     },
+    {
+      key: 'SkillManifest',
+      name: '',
+      className: 'skillManifest',
+      fieldName: 'skillManifestUrl',
+      minWidth: 114,
+      maxWidth: 134,
+      data: 'string',
+      onRender: (item: BotStatus) => {
+        return (
+          item?.skillManifestUrl && (
+            <ActionButton
+              title={item.skillManifestUrl}
+              onClick={() => {
+                copyStringToClipboard(item.skillManifestUrl);
```

Comment:
seems like item.skillManifestUrl will always have a value, why is it optional in the copyStringToClipboard function?

Reply:
The manifest URL will be empty in many cases.  Root bots won't have one and you have to generate on after adding a skill.

Reply:
you are rendering conditionally

Reply:
The compiler complains because the onClick is a callback.